client/my-sites/themes/thanks-modal.jsx

```diff
@@ -156,14 +170,16 @@ class ThanksModal extends Component {
 
 	renderContent = () => {
 		const { name: themeName, author: themeAuthor } = this.props.currentTheme;
+		const { isUsingClassicEditor, shouldEditHomepageWithGutenberg } = this.props;
+		const shouldSwitchEditors = isUsingClassicEditor && shouldEditHomepageWithGutenberg;
```

Comment:
Probably we should also change `shouldEditHomepageWithGutenberg` here with `isThemeGutenbergFirst` too